pkg/cmd/cli/restic/server_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2018 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package restic
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/heptio/velero/pkg/test"
+)
+
+func Test_validatePodVolumesHostPath(t *testing.T) {
```

Comment:
TestValidatePodVolumesHostPath

Reply:
So, when we use a generated test from VScode, that's how it generates a test name for private functions. I sorta like having the ability to distinguish tests for exported vs non-exported functions. Could we adopt this style?

Reply:
I'm open to it if the team prefers it in general

Reply:
I'm not opposed.